docs/.vitepress/config.mts

```diff
@@ -16,10 +16,8 @@ export default withMermaid({
     themeConfig: {
         // https://vitepress.dev/reference/default-theme-config
         nav: [
-            {text: 'Home', link: '/'},
             {text: 'About', link: '/about/'},
-            {text: 'Features', link: '/features/'},
-            {text: 'Guides', link: PATH_GUIDES},
+            {text: 'Services', link: '/features/'},
```

Comment:
sollte der pfad / der ordner auch noch zu servicesumbenannt werden?

Reply:
ich würde auch die Dateien verschieben. Das macht die Wartung leichter.

Reply:
siehe commit [a109f47](https://github.com/it-at-m/Wahllokalsystem/pull/495/commits/a109f478e0698f1fe97a33e8dd92fba7178a6011)